src/routes/th/index.ts

```diff
@@ -0,0 +1,13 @@
+import { Request, Response, Router } from "express";
+
+let tamuHackRouter = Router();
+
+tamuHackRouter.get('/', async (req: Request, res: Response) => {
```

Comment:
Why does this one redirect to `/2019` whereas the HH index route doesn't?

Reply:
Howdyhack router now does